run.sh

```diff
@@ -346,6 +349,23 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  if [[ ! "${REQUIRE-}" ]]; then
+    h3 "No package dependencies listed"
+    STATUS SKIP
+    return
+  fi
+
+  h3 "Installing composer packages"
+  touch /app/composer.json
```

Comment:
No strong feelings on it. I just think it would be easier from a maintenance perspective.

Additionally, going this way, you wouldn't need to set `REQUIRE`. Just have the run script check for the existence of `composer.json`.

I'll let @karellm be the judge on which way to go for that.